### PR TITLE
include EasyBuild v2.0.0 release notes

### PR DESCRIPTION
**Pull request by boegel**
rendered version at http://boegel-eb.readthedocs.org/en/eb20_release_notes/Release_notes.html

@fgeorgatos, @wpoely86, @rjeschmi: please review

### CODE REVIEW

File: docs/Release_notes.rst [hunk 1]
```diff
@@ -3,9 +3,151 @@
 EasyBuild release notes
 =======================
 
-The latest version of EasyBuild provides support for building and installing **551** different software packages,
-using 32 different (compilers) *toolchains*. It contains 141 software-specific easyblocks and 20 generic *easyblocks*,
-alongside 3,066 *easyconfig files*.
+The latest version of EasyBuild provides support for building and installing **581** different software packages,
+using 32 different (compilers) *toolchains*. It contains 143 software-specific easyblocks and 21 generic *easyblocks*,
```
Comment (wpoely86) [inline docs/Release_notes.rst]: be consistent here, some thing are italic, some not

Comment (boegel) [inline docs/Release_notes.rst]: not sure why I'm using italics at all, so I'll just drop them

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Comment by boegel** [comment]
contents have already been reviewed via https://github.com/hpcugent/easybuild-framework/pull/1208, https://github.com/hpcugent/easybuild-easyblocks/pull/572, https://github.com/hpcugent/easybuild-easyconfigs/pull/1453

formatting is OK, so good to go in

**Pull request merged**